Feat/cleanup

FlorianZipser opened the pull request:
This branch contains some code cleanup. But there is still more work to do. This pull request is just to synchronize with develop and will be used for further clean up.

Commit:
some reorganization

Commit:
reorganized packages, added new test for multiple documents in one file

Commit:
simplified call of deserialize() aka load(...)

Commit:
more clean up

Commit:
simplified extraction of attributes

Commit:
more cleanup

Commit:
removed unnecessary PublicMapper classes

Commit:
Revert "more cleanup"

This reverts commit 21ba868687ca89d2f816ab24f24eac386766433f.

Commit:
Revert "simplified extraction of attributes"

This reverts commit 0e2bd041816685cf63368ad0f69f9c828741f1b5.

Commit:
Revert "Revert "simplified extraction of attributes""

This reverts commit 23a21f568914bd8768705d3fdfe3e333c5ebab23.

Commit:
removed unnecessary parameter in addMetaAnnotation

Commit:
more cleanup

FlorianZipser committed:
added bracket to condition

FlorianZipser committed:
Merge branch 'develop' into feat/cleanup

Conflicts:
	src/test/java/org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java

FlorianZipser committed:
Merge branch 'develop' into feat/cleanup

Conflicts:
	src/test/java/org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java

thomaskrause commented:
I get a compilation error about a missing class "TabReader". Maybe you forgot to add the file?

```
-------------------------------------------------------------
COMPILATION ERROR : 
-------------------------------------------------------------
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[35,9] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[39,16] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[86,50] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[98,64] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[100,64] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[102,64] error: cannot find symbol
 class TabReaderTest
org/corpus_tools/peppermodules/treetagger/model/resources/TabReaderTest.java:[104,64] error: cannot find symbol
7 errors 
```

FlorianZipser committed:
Merge branch 'feat/cleanup' of https://github.com/korpling/pepperModules-TreetaggerModules into feat/cleanup

FlorianZipser committed:
# WARNING: head commit changed in the meantime

Merge branch 'feat/cleanup' of
https://github.com/korpling/pepperModules-TreetaggerModules.git into
feat/cleanup

FlorianZipser commented:
I am sorry you were right. It seems that my branch wasn't in sync. I believe it is fixed now.
